server_tests/test/clientTests.js

```diff
@@ -1337,7 +1337,7 @@ describe("RUST API TEST SUITE", function() {
             return Utility.restoreDefaultOptions(defaultOptionsFile)
             .then(function() {
                 return Utility.deleteAllImages();
-            });
+            }, wrapError);
```

Comment:
This isn't enough because `deleteAllImages` also needs error wrapping. Probably better to use `.catch(wrapError)`, we should also check all other uses of these functions...